### DISCUSSION TITLE
Organizations with not lowercased username are not displaying

### DISCUSSION THREAD

**Discussion by hatamiarash7** [yhype/yhype-feedback#7] [Q&A] [answered]
Hi <img src="https://media.giphy.com/media/hvRJCLFzcasrR4ia7z/giphy.gif" width="16">

How can I install **Ÿ HŸPE** in my organizations? I couldn't find any settings to get permissions like other applications

**Comment by antonkomarev** [comment]
@hatamiarash7 Very good question. I need to add it to the FAQ.

You have to be visible member of the Organization to let us see you, otherwise GitHub doesn't provide us your membership 😢 And you should have admin rights for the Organization to be able to install App in it.

**Reply by hatamiarash7** [reply]
I'm Admin and visible ... But I can't find any settings to give Ÿ HŸPE the permissions to access Org.

**Reply by antonkomarev** [reply]
You could try to go to this URL and add it manually: https://github.com/organizations/{ORG_NAME}/settings/installations/12728107
Just replace {ORG_NAME} with name of your organization.

**Reply by hatamiarash7** [reply]
It's 404 ... !

Currently, I can see my Orgs in the dashboard :

![image](https://user-images.githubusercontent.com/16325641/162568894-00d2fa54-230d-4934-82de-1a71086b7c16.png)

But I can't view them. `Can not see other users personal accounts `

**Reply by antonkomarev** [reply]
You may write me in [Discord](https://discord.gg/nFBSdHFyqm) and we will try to debug this issue.

**Reply by boredland** [reply]
I've got the same problem :-(

**Reply by antonkomarev** [reply]
@boredland Just deployed a fix, you could try again.

**Reply by boredland** [reply]
Na, still "Can not see other users personal accounts". Just realized that my installation ID differs though?
https://github.com/organizations/Manjaro-Sway/settings/installations/23388497

**Reply by antonkomarev** [reply]
Working on it. Will notify when it will be completely ready. Right now you could write upper case name in URL to make it work.

**Reply by hatamiarash7** [reply]
Another question. What is the current process of reviewing visits to Orgs? The installation instruction said we should add an image in the personal readme repo. What should we do for Orgs?

**Reply by hatamiarash7** [reply]
> Another question. What is the current process of reviewing visits to Orgs? The installation instruction said we should add an image in the personal readme repo. What should we do for Orgs?

Do you have any idea? @antonkomarev

**Reply by antonkomarev** [reply]
@hatamiarash7 you want to add pixel for Org Profile? I have some code preparations but not much free time to complete it.

**Reply by hatamiarash7** [reply]
> @hatamiarash7 you want to add pixel for Org Profile? I have some code preparations but not much free time to complete it.

Exactly !
At the moment, Every Org can have a Readme repo ( like normal users ). We can use your app to monitor Orgs too.

**Reply by hatamiarash7** [reply]
Take a look at [this](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file)

**Reply by antonkomarev** [reply]
Backend is working at this moment, but there is no interface to display it. Need to make some frontend work.

**Reply by antonkomarev** [reply]
@hatamiarash7 you can vote here:
- #8

**Comment by antonkomarev** [comment, marked as answer]
Thanks to the @hatamiarash7, fix has been deployed. Now organization search is case-insensitive.

@boredland you could re-check if it works for you too.

**Reply by boredland** [reply]
Works like a charm!

**Comment by antonkomarev** [comment]
Rule of thumb. Never modify the values in the class constructor 💔 Stupid mistake.
